internal/controllers/ping.go

```diff
@@ -1,26 +1,52 @@
 package controllers
 
 import (
+	"net/http"
+
 	"github.com/NodeFactoryIo/vedran/internal/auth"
 	"github.com/NodeFactoryIo/vedran/internal/models"
 	log "github.com/sirupsen/logrus"
-	"net/http"
+)
+
+const (
+	pingIntervalSeconds = 10
```

Comment:
Maybe move to a separate file that will just hold constants

Reply:
I would maybe keep it here